package.json

```diff
@@ -31,7 +31,7 @@
   "scripts": {
     "pretest": "npm run lint",
     "prepublishOnly": "npm run build",
-    "lint": "tslint --project . --fix",
+    "lint": "eslint --fix *.ts",
```

Comment:
This glob doesn't look like it catches test files—is that correct?

Would `eslint --fix .` work?

Reply:
Nice catch, `eslint --fix .` sadly doesn't do anything, so I opted for `eslint --fix '**/*.ts'` instead.